lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -513,80 +553,82 @@ def _get_gps_status_as_bool(enodeb: EnodebAcsStateMachine) -> bool:
 
 
 def _get_and_cache_gps_coords(enodeb: EnodebAcsStateMachine) -> Tuple[
-    str, str,
+    str, str, str,
 ]:
     """
     Read the GPS coordinates of the enB from its configuration or the
     cached coordinate file if the preceding read fails. If reading from
     enB configuration succeeds, this method will cache the new coordinates.
 
     Returns:
-        (str, str): GPS latitude, GPS longitude
+        (str, str, str): GPS latitude, GPS longitude
```

Comment:
update docs here

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812281-3fc3fca6-3524-4d1f-b416-f07876bb3292.png)
modified, thank you!